docs/modules/ROOT/pages/csharp-client-getting-started.adoc

```diff
@@ -5,118 +5,326 @@
 :page-categories: Get Started
 :page-lang: csharp
 :page-enterprise:
-:page-est-time: 15mins.
-:description: In this tutorial you will see how to connect a .NET client to Hazelcast and manipulate a map.
+:page-est-time: 5-10 mins
+:description: In this tutorial you will see how to connect with the Hazelcast .NET client and manipulate a map.
 
-{description}
-
-// Give some context about the use case for this tutorial. What will the reader learn?
-== Context
+== What You'll Learn
 
-A quick start to use Hazelcast with .NET Client.
+{description}
 
 == Before you Begin
 
 Before starting this tutorial, make sure the following prerequisites are met:
 
 
 * .NET SDK 6.0 or above
-* Viridian account or Docker
+* https://hazelcast.com/products/viridian/[Hazelcast Viridian Cloud Account]
 * An IDE
 
-== Step 1. Start Hazelcast
+== Start a Hazelcast Viridian Cloud Cluster
 
-First, run the latest Hazelcast instance with Docker.
+1. Sign up for a Hazelcast Viridian Cloud account (Free accounts are available).
+2. Log in to your Hazelcast Viridian Cloud account and navigate to your https://viridian.hazelcast.com[cluster list].
+3. Create a new cluster or start an existing one.
 
-```
-docker run --name hazelcast -p 5701:5701 hazelcast/hazelcast
-```
+== Setup a Hazelcast Client
 
-If you don’t have Docker or don’t want to use it, you can connect to https://docs.hazelcast.com/cloud/connect-to-cluster[Viridian].
+1. Use the quick connection guide to download the preconfigured .NET sample client.
+2. Follow the instructions on the quick connection guide to start using your cluster.
 
-== Step 2. Create a .NET Application
+To understand and use the client, review the https://hazelcast.github.io/hazelcast-csharp-client/versions.html[.NET API documentation] to better understand what is possible.
 
-Next, in a folder run the command and create a new .NET console application.
+== Understanding the .NET Client
+
+The following section creates and starts a Hazelcast client with default configuration, before shutting it down when the Program exits.
+
+[source,cs]
+----
+using Hazelcast;
 
-```
-dotnet new console -n HazelcastQuickStart --use-program-main
-```
+using System;
+using System.Threading.Tasks;
 
-Then, go to the project folder and add a dependency to the Hazelcast.Net NuGet package.
+namespace Client
+{
+    internal static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            // Create a client connection            
+            var options = new HazelcastOptionsBuilder()
+                .With(args) // Pass command line args to the client
```

Comment:
It looks nice. I only have a comment. Since we put Viridian forward, I believe we should include the configuration for cloud connection too. Otherwise, user should figure out how to connect in a starting guide. WDYT?
```suggestion
                .With(args) // Pass command line args to the client
                .With(config =>
                {
                    // Your Viridian cluster name.
                    config.ClusterName = "CLUSTER_NAME";
                    // Your discovery token to connect Viridian cluster.
                    config.Networking.Cloud.DiscoveryToken = "YOUR_DISCOVERY_TOKEN";
                    // Enable metrics to see on Management Center.
                    config.Metrics.Enabled = true;
                    // Configure SSL.
                    config.Networking.Ssl.Enabled = true;
                    config.Networking.Ssl.ValidateCertificateChain = false;
                    config.Networking.Ssl.CertificatePath = "PATH_TO_CLIENT.PFX";
                    config.Networking.Ssl.CertificatePassword = "CERTIFICATE_PASSWORD";
                })
                .Build();
```

Reply:
I think you are right.